README.md

```diff
@@ -1,2 +1,69 @@
-# windup-operator
-Windup Operator for Openshift-Kubernetes
+# windup-operator project
+
+This project creates an Operator to deploy Windup/MTA on an Openshift cluster ( Kubernetes one in future releases)
+
+It consists on a series of yaml files to deploy the operator (CRD, role, rolebinding, serviceaccount) and the native artifact compiled using Quarkus and GraalVM
+
+At this moment the operator reacts to creation of the Windup Custom Resource and will deploy all needed objects ( deployments, ingress, services, persistent volumes)
+
+## Building and pushing the Java code manually in native mode
+1. The following command will use the configuration in the `application.properties` file :  
+```
+quarkus.kubernetes.deployment-target=kubernetes, openshift
+quarkus.native.resources.includes=k8s/def/*.yaml
+quarkus.container-image.registry=quay.io
+quarkus.container-image.group=windupeng
+quarkus.container-image.name=windup-operator-native
+quarkus.container-image.tag=latest
+quarkus.kubernetes.service-type=load-balancer
+
+```
+1. Install JDK 11  
+You can use different ways , but SDKMan is very easy
+2. Execute the maven command:  
+`mvn clean package -Pnative -Dquarkus.native.container-build=true -Dquarkus.container-image.push=true`
+
+## Installation
+
+1. We can log in the Openshift cluster using `oc login .....` . You will need a user with cluster-wide permissions to deploy the CRD.
+1. Move to the `src/main/resources` folder
+1. If you are installing the Operator on a cluster without the `mta` namespace , you first should create the namespace with  
```

Comment:
With the latest changes this step has become optional so worth letting the user know s/he can use any namespace. If none available, then this command will create one for her/him

Reply:
This command will not create any namespace..... you need to create a namespace or use one of your election.
These scripts are for convenience and are considering `mta` as the name space. If you want to install the operator in another unexisting namespace you need to create the namespace before, and then change these scripts to use that namespace.

Reply:
> This command will not create any namespace..... 

Well, I just wrote this command creates a namespace because it's what I read in this installation step:
_If you are installing the Operator on a cluster without the `mta` namespace , you first should create the namespace with
`oc apply -f windup.namespace.yaml`_

I think it would be worth adding a short note that installation on a different namespace (not named `mta`) requires changes in all the scripts.

Reply:
:+1: